code/modules/vehicles/cars/clowncar.dm

```diff
@@ -95,8 +94,8 @@
 	to_chat(user, span_danger("You use the [banana] to repair [src]!"))
 	qdel(banana)
 
-/obj/vehicle/sealed/car/clowncar/proc/check_bumped(atom/bumped)
```

Comment:
the first arg of a signal is source, it's probably not working because bumped is meant to be the second arg here.